pollination/honeybee_vtk/export_images.py

```diff
@@ -0,0 +1,79 @@
+from dataclasses import dataclass
+from pollination_dsl.function import Function, command, Inputs, Outputs
+from pydantic.typing import IntStr
+
+
+@dataclass
+class ExportImages(Function):
+    """Export images from am hbjson file."""
+
+    hbjson_file = Inputs.file(
+        description='HBJSON file.',
+        path='input.hbjson'
+    )
+
+    name = Inputs.str(
+        description='Name of image files.',
+        default='Camera'
+    )
+
+    folder = Inputs.folder(
+        description='Target folder where the images will be written.',
+        path='output_folder',
+        default='.'
+    )
```

Comment:
You don't need to expose this output to user. in the context of the function you want to control where the generated files are to copy them back as needed.

Reply:
I actually saw that you are not really using it in the command. You only need to remove it from here.

Reply:
removed